site/docs/windows.md

```diff
@@ -106,6 +110,10 @@ To build C++ targets, you need:
     or
 
     ```
+    rem This is for the Visual C++ Build Tools 2015.
+    set BAZEL_VC=C:\Program Files (x86)\Microsoft Visual Studio 14.0
+    
+    rem This is for the Visual Studio 2017 Build Tools.
```

Comment:
Let's not recommend the 2017 version, not even in examples, until issue #3949 is closed.

Reply:
Fixed! :)